airflow/providers/common/sql/hooks/sql.py

```diff
@@ -114,6 +114,7 @@ def __init__(self, *args, schema: str | None = None, log_sql: bool = True, **kwa
         # Hook deriving from the DBApiHook to still have access to the field in it's constructor
         self.__schema = schema
         self.log_sql = log_sql
+        self.query_ids: list[str] = []
```

Comment:
should this be `running_query_ids`?

Reply:
This name was already used in Snowflake so I just generalized it.
Let's rename it

Reply:
I understand, but we don't need to stick with the old naming, especially when adding to new base hook.  I think it's good to specify what kind of query ids. but in this case they aren't actually running queries so it's in some ways a moot point.

Reply:
Okay then, renamed it